public/examples/ach.html

```diff
@@ -137,33 +138,24 @@
           event.preventDefault();
 
           try {
-            // disable the submit button as we await tokenization and make a payment request.
-            cardButton.disabled = true;
             achButton.disabled = true;
-            const token = await tokenize(paymentMethod, options);
-            const paymentResults = await createPayment(token);
-            displayPaymentResults('SUCCESS');
-
-            console.debug('Payment Success', paymentResults);
+            tokenize(paymentMethod, options);
```

Comment:
I know ACH relies mostly on the event listener, but I feel like this should still be awaited..?

Reply:
All the errors will now be captured via the listener so we don't need to await this anymore.

Reply:
In that case, the async keyword should be removed from the tokenize() method definition on line 50.